src/controllers/check.your.answers.controller.ts

```diff
@@ -48,25 +46,30 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
 
 export const post = async (req: Request, res: Response, next: NextFunction) => {
   try {
-    const session = req.session as Session;
     logger.debugRequest(req, `POST ${config.CHECK_YOUR_ANSWERS_PAGE}`);
 
+    const session = req.session as Session;
+    const appData: ApplicationData = getApplicationData(session);
+
     if (isActiveFeature(config.FEATURE_FLAG_ENABLE_REFRESH_TOKEN_29092022)) {
       const accessToken = await refreshToken(req, session);
       logger.infoRequest(req, `New access token: ${accessToken}`);
     }
 
-    const transaction: Transaction = await postTransaction(req, session);
-    logger.infoRequest(req, `Transaction created, ID: ${transaction.id}`);
+    const isSaveAndResumeFeatureActive = isActiveFeature(config.FEATURE_FLAG_ENABLE_SAVE_AND_RESUME_17102022);
+    const transactionID = (!isSaveAndResumeFeatureActive)
+      ? await postTransaction(req, session)
+      : appData.transaction_id as string;
 
-    const overseaEntity: OverseasEntityCreated = await createOverseasEntity(req, session, transaction.id as string);
-    logger.infoRequest(req, `Overseas Entity Created, ID: ${overseaEntity.id}`);
+    const overseaEntityID = (!isSaveAndResumeFeatureActive)
+      ? await createOverseasEntity(req, session, transactionID)
+      : appData.overseas_entity_id as string;
```

Comment:
we'll need to do a PUT to the OE API if feature flag is active as the only data that will have been sent so far is the presenter data (once PUT endpoint is done)

Reply:
Yep, it is part of [ROE-1407](https://companieshouse.atlassian.net/browse/ROE-1407)